packages/features/insights/filters/OrgTeamsFilter.tsx

```diff
@@ -126,35 +130,32 @@ export const OrgTeamsFilter = () => {
           label={t("yours")}
         />
 
-        <Divider />
-        {(data || [])
-          .filter((team) => !team.isOrg)
-          .filter((team) => team.name?.toLowerCase().includes(query.toLowerCase()))
-          .map((team) => {
-            return (
-              <FilterCheckboxField
-                testId="org-teams-filter-item"
-                key={team.id}
-                id={team.name || ""}
-                label={team.name || ""}
-                checked={selectedTeamId === team.id && orgTeamsType === "team"}
-                onChange={(e) => {
-                  if (e.target.checked) {
-                    onSelected({ type: "team", teamId: team.id });
-                  } else if (!e.target.checked) {
-                    resetSelection();
-                  }
-                }}
-                icon={
-                  <Avatar
-                    alt={team.name || ""}
-                    imageSrc={getPlaceholderAvatar(team.logoUrl, team.name)}
-                    size="xs"
-                  />
+        {teams.length > 0 && <Divider />}
```

Comment:
these are mostly indentation change. 

turn this on for better diff:
![Screenshot 2025-02-11 at 11 25 08](https://github.com/user-attachments/assets/60684d29-a086-411b-8392-55183ff163c7)